### ISSUE TITLE
Form validation error not handled gracefully when creating new voucher

### ISSUE THREAD

**Issue by TomDeSmet** [verbb/gift-voucher#97]
### Describe the bug

When creating a new voucher and not filling in the SKU and price, we get an error window instead of just a notification inside the CP that we didn't fill in the required forms.

### Steps to reproduce

1. Add a new voucher and don't add a SKU or price.
2. Save the voucher.

### Craft CMS version

4.2.5.2

### Plugin version

3.0.0

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/gift-voucher` requirement in `composer.json` to:

```
"require": {
  "verbb/gift-voucher": "dev-craft-4 as 3.0.0",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [3.0.1](https://github.com/verbb/gift-voucher/releases/tag/3.0.1)
